HW_5_PandaVis/Script_for_review.py

```diff
@@ -0,0 +1,320 @@
+import os.path
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib.patches import ConnectionPatch
+import seaborn as sns
+
+### Task 1, rRNAs
+
+def read_gff(file: str) -> pd.DataFrame:
+    cols = ["chromosome", "source", "type", "start", "end", "score", "strand", "phase", "attributes"]
+    df = pd.read_csv(file, sep='\t', skiprows=1, header=None)

```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    df = pd.read_csv(file, sep='\t', comment="#", header=None)

```